ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -37,6 +37,13 @@ if [[ ${ZONES_COUNT} -gt ${MAX_ZONES_COUNT} ]]; then
   exit 1
 fi
 
+MASTER_REPLICAS=3
+if [[ "${SIZE_VARIANT}" == "compact" ]]; then 
```

Comment:
Shouldn't we also use smaller flavors when size variant is compact, similar to what AWS, GCP and Azure do?
Perhaps in a later commit.

Reply:
to be defined later, when the job will work. I agree this should be revisited, just not now.